examples/games/breakout.rs

```diff
@@ -420,3 +437,111 @@ fn play_collision_sound(
+    }
+}
+
+fn stepping_ui(
+    mut commands: Commands,
+    schedules: Res<Schedules>,
+    mut ui: Query<(Entity, &mut Text, &Visibility), With<SteppingUi>>,
+    mut schedule_events: EventReader<bevy::ecs::schedule::ScheduleEvent>,
+) {
+    let label = Box::new(CoreSchedule::FixedUpdate);
+    let fixed = schedules
+        .get(&*label)
+        .expect("Schedules resource should contain CoreSchedule::FixedUpdate");
+    let (entity, mut text, vis) = ui.single_mut();
+
+    if !fixed.stepping() {
+        if vis == Visibility::Inherited {
+            commands.entity(entity).insert(Visibility::Hidden);
+        }
+        return;
+    }
+
+    if vis == Visibility::Hidden {
+        commands.entity(entity).insert(Visibility::Inherited);
+    }
+    text.sections[3].value = fixed.next_system().unwrap_or("unknown".to_string());
+
+    for ev in schedule_events.iter() {
+        use bevy::ecs::schedule::ScheduleEvent::*;
+        match ev {
+            StepFrame(_) => text.sections[1].value = "stepped frame".to_string(),
+            StepSystem(_) => {
+                text.sections[1].value = format!(
+                    "ran {}",
+                    fixed.next_system().unwrap_or("unknown".to_string())
+                )
+            }
+            _ => text.sections[1].value = "enabled stepping".to_string(),
+        }
+    }
+}
+
+const STEPPING_FONT_SIZE: f32 = 24.0;
+const STEPPING_FONT_COLOR: Color = Color::rgb(0.2, 0.2, 0.2);
+
+fn setup_stepping(mut commands: Commands, asset_server: Res<AssetServer>) {
+    commands.spawn((
+        SteppingUi,
+        TextBundle::from_sections([
+            TextSection::new(
+                "Stepping: ",
+                TextStyle {
+                    font: asset_server.load("fonts/FiraSans-Bold.ttf"),
+                    font_size: STEPPING_FONT_SIZE,
+                    color: STEPPING_FONT_COLOR,
+                },
+            ),
+            TextSection::from_style(TextStyle {
+                font: asset_server.load("fonts/FiraMono-Medium.ttf"),
+                font_size: STEPPING_FONT_SIZE,
+                color: STEPPING_FONT_COLOR,
+            }),
+            TextSection::new(
+                "\nNext system: ",
+                TextStyle {
+                    font: asset_server.load("fonts/FiraSans-Bold.ttf"),
+                    font_size: STEPPING_FONT_SIZE,
+                    color: STEPPING_FONT_COLOR,
+                },
+            ),
+            TextSection::from_style(TextStyle {
+                font: asset_server.load("fonts/FiraMono-Medium.ttf"),
+                font_size: STEPPING_FONT_SIZE,
+                color: STEPPING_FONT_COLOR,
+            }),
+        ])
+        .with_style(Style {
+            position_type: PositionType::Absolute,
+            position: UiRect {
+                top: Val::Px(5.0),
+                right: Val::Px(200.0),
+                ..default()
+            },
+            ..default()
+        }),
+    ));
```

Comment:
This is a terrible UI; any and all improvements accepted.

Honestly, I wish an example could pull in an entity inspector and demonstrate full view & edit of components between system execution.

Reply:
The most important fix IMO is to keep the text position constant. I think that simply increasing the size of the parent node should do the trick?

Reply:
I found a fix, setting `position.left` resolved it.  None of the size or alignment properties seemed to affect it.